src/components/02-molecules/Header/index.js

```diff
@@ -1,13 +1,16 @@
 import React, { PropTypes } from 'react';
+import { browserHistory } from 'react-router';
 import styles from './styles.scss';
 
+
 const Header = ({ user, logout }) => (
   <header className={styles.header}>
     <span className={styles.hello}>Hello</span>
     <span className={styles.name}>
       { user.name }!
     </span>
-    <span className={styles.logout} onClick={logout}>Log Out</span>
+    <span className={styles.link} onClick={() => browserHistory.push('/admin')}>Admin</span>
```

Comment:
@monksp-buildit Are you in a design systems meeting or something? :)

Yes, completely agree. We talked about that, but decided to make it a separate piece of work since there are many other places where we should be using a `<Link />` instead of this raw stuff.